internal/apis/certmanager/types_issuer.go

```diff
@@ -259,6 +259,10 @@ type VaultKubernetesAuth struct {
 	// A required field containing the Vault Role to assume. A Role binds a
 	// Kubernetes ServiceAccount with a set of Vault policies.
 	Role string
+
+	// List of audiences to include as an audience claim in the token created
+	// +optional
+	TokenAudiences []string
```

Comment:
suggestion: I think this comment isn't clear that we're appending to the default claim. Looking at this I think someone could reasonably think that the list we pass here is the exact list that will be present in the JWT.

Could we reword this to make it clearer? (Same applies to every other instance of TokenAudiences)

```suggestion
	// TokenAudiences is an option list of extra audiences to include in the token passed to Vault. The default token
	// consisting of the issuer's namespace and name is always included.
	// +optional
	TokenAudiences []string
```